rllib/utils/framework.py

```diff
@@ -94,6 +100,12 @@ def get_device(config: "AlgorithmConfig", num_gpus_requested: int = 1):
                 # `torch.cuda.device_count() = 1` and torch.device(0) maps to that GPU
                 # with ID=1 on the node.
                 return torch.device(config.local_gpu_idx)
+        elif custom_resources_requested:
+            from ray.air._internal.torch_utils import get_devices
+
+            # The `get_devices()` api in ray.air should handle the custom accelerator
+            # and return torch.device("cpu") if not accelerator is available.
+            return get_devices()[0]
```

Comment:
What if there are multiple custom resources on a single node? Which `Learner` gets assigned to which of the resources? We handle this case for GPUs with the `local_gpu_idx` which each `Learner` actor receives in its order of initialization.